images/linux/scripts/helpers/Common.Helpers.psm1

```diff
@@ -54,4 +54,16 @@ function Get-AndroidPackages {
     $androidSDKManagerPath = "/usr/local/lib/android/sdk/tools/bin/sdkmanager"
     $androidPackages = & $androidSDKManagerPath --list --verbose
     return $androidPackages
+}
+
+function Get-EnvironmentVariable($variable) {
+    return [System.Environment]::GetEnvironmentVariable($variable)
+}
+
+function Update-Environment {
+    $variables = Get-Content "/etc/environment" | Select-String -NotMatch "^#"
+    $variables | ForEach-Object { 
+        $variable = $_[0].ToString().split("=")
+        [System.Environment]::SetEnvironmentVariable($variable[0], $variable[1])
```

Comment:
What is the default value for `EnvironmentVariableTarget`? (third parameter)/
I think we should specify it explicitly as `Process` to make sure that these variables are not saved somewhere